server/chat-commands/room-settings.ts

```diff
@@ -1454,3 +1503,48 @@ export const roomSettings: SettingsHandler[] = [
 		],
 	}),
 ];
+
+export const pages: PageTable = {
+	permissions(args, user, connection) {
+		let [room, rank] = args.filter(Boolean) as [string, string | undefined];
+		this.title = `[Permissions]`;
+		this.extractRoom();
+		const u = (str: string) => {
+			const char = str.charAt(0).toUpperCase();
+			str = str.slice(1);
+			return char + str;
+		};
+		if (!room) return `<h2>This room does not exist or does not support permissions.</h2>`;
+		if (!user.authAtLeast('%', this.room)) return `<h2>Access denied.</h2>`;
+		if (!rank) {
+			let buf = `<div class="pad"><center><strong>Groups on ${this.room.title}:</strong><hr/ >`;
+			for (const r of Config.groupsranking.slice(2)) {
+				if (Config.groups[r]?.globalonly) continue;
+				const name = (Config.groups[r] ? Config.groups[r].name : r).replace(/ /, '');
+				buf += `<a href="/view-permissions-${room}-${Dashycode.encode(name)}" `;
```

Comment:
These buttons are pointless and just make it harder to see what permissions exist. They should all be on one page.